discord/channel.py

```diff
@@ -245,6 +245,8 @@ def purge(self, *, limit=100, check=None, before=None, after=None, around=None,
             Same as ``reverse`` in :meth:`history`.
         reason: Optional[str]
             The reason for doing this action. Shows up on the audit log.
+        no_bulk: bool
+            If True, never use bulk delete.
```

Comment:
Swap the `True` -> `False` with the `bulk` change and then probably state that pretty much the only use case is for deleting your own messages due to the way bulk delete works Discord side.